ocaml/idl/datamodel_vm.ml

```diff
@@ -214,17 +214,14 @@ let power_behaviour =
   let snapshot_with_quiesce = call
       ~name:"snapshot_with_quiesce"
       ~in_product_since: rel_orlando
-      ~doc:"Snapshots the specified VM with quiesce, making a new VM. Snapshot automatically exploits the capabilities of the underlying storage repository in which the VM's disk images are stored (e.g. Copy on Write)."
+      ~lifecycle:[ Published, rel_orlando, ""; Removed, rel_quebec, "" ]
+      ~doc:"Support for this feature has been removed. Snapshots the specified VM with quiesce, making a new VM. Snapshot automatically exploits the capabilities of the underlying storage repository in which the VM's disk images are stored (e.g. Copy on Write)."
       ~result: (Ref _vm, "The reference of the newly created VM.")
       ~params:[
         Ref _vm, "vm", "The VM to be snapshotted";
         String, "new_name", "The name of the snapshotted VM"
       ]
-      ~errs:[Api_errors.vm_bad_power_state; Api_errors.sr_full; Api_errors.operation_not_allowed;
-             Api_errors.vm_snapshot_with_quiesce_failed;
-             Api_errors.vm_snapshot_with_quiesce_timeout;
-             Api_errors.vm_snapshot_with_quiesce_plugin_does_not_respond;
-             Api_errors.vm_snapshot_with_quiesce_not_supported ]
+      ~errs:[ Api_errors.vm_snapshot_with_quiesce_not_supported ]
```

Comment:
@robhoes  should i create a new api error to use here? (since otherwise the semantics of the error will change)

Reply:
I'd just leave the old list of errors here, to document what the function used to do.